src/storage/clickhouse/clickhouse.service.ts

```diff
@@ -163,14 +165,16 @@ export class ClickhouseService implements OnModuleInit {
           .insert(
             'INSERT INTO stats.validator_attestations ' +
               '(start_fetch_time, validator_pubkey, validator_id, committee_index, committee_length, committees_at_slot, ' +
-              'validator_committee_index, slot_to_attestation, attested, info_from_block, nos_id, nos_name) VALUES',
+              'validator_committee_index, slot_to_attestation, attested, inclusion_delay, valid_head, valid_target, ' +
+              'valid_source, info_from_block, nos_id, nos_name) VALUES',
           )
           .stream();
         for (const a of chunk) {
           await ws.writeRow(
             `(${slotTime}, '${a.pubkey || ''}', '${a.validator_index || ''}', ${parseInt(a.committee_index)}, ` +
               `${parseInt(a.committee_length)}, ${parseInt(a.committees_at_slot)}, ${parseInt(a.validator_committee_index)}, ` +
-              `${parseInt(a.slot)}, ${a.attested ? 1 : 0}, ${a.in_block ? parseInt(a.in_block) : 'NULL'}, ` +
+              `${parseInt(a.slot)}, ${a.attested}, ${a.inclusion_delay ?? 'NULL'}, ${a.valid_head ?? 'NULL'}, ` +
```

Comment:
To set `DEFAULT NULL` or smth in migration below and do not use explicit `NULL`?

Reply:
It has to be done either everywhere or nowhere. I think to do it when writing unit tests

Reply:
I see no reason for all or nothing. New one migration and you can start do things in the correct way.

Reply:
In current moment, migration creates `Nullable` with default `NULL`. With current recording approach, insert query string must present values for all columns and they cannot be `undefined`. What you suggest will be possible to do with some ORM or additional insertion logic for current approach in the future. That's why I said it should either be done everywhere or nowhere. This is beyond the scope of this PR
Left TODO